modules/persistenceService/src/main/scala/connectionLayer/DbQueries.scala

```diff
@@ -0,0 +1,35 @@
+package connectionLayer
+
+import persistenceAlgebra.DbAlgebra
+import persistenceModel.{Id, Name, User, UserName}
+import doobie._
+import doobie.implicits._
+import cats.effect.Bracket
+import doobie.postgres.implicits._
+
+import java.util.UUID
+
+class DbQueries[F[_]](implicit ev: Bracket[F, Throwable]) extends DbAlgebra {
+  def insert(user: User): ConnectionIO[Int] = DbQueries.insert(user).run
+
+  def find(username: UserName): ConnectionIO[Option[User]] = DbQueries.find(username).option
+
+  def remove(user: UserName): ConnectionIO[Int] = DbQueries.remove(user).run
+}
+
+object DbQueries {
+
+  implicit val nameMeta         = Meta.StringMeta.timap[Name](Name(_))(name => name.name)
+  implicit val userName         = Meta[String].timap[UserName](UserName(_))(userName => userName.username)
+  implicit val idMeta: Meta[Id] = Meta[UUID].timap(uuid => Id(uuid))(id => id.id)
```

Comment:
Just to be consistent we could write
```suggestion
  implicit val nameMeta: Meta[Name]      = Meta.StringMeta.timap[Name](Name(_))(_.name)
  implicit val userName: Meta[UserName]  = Meta[String].timap[UserName](UserName(_))(_.username)
  implicit val idMeta: Meta[Id]          = Meta[UUID].timap(Id(_))(_.id)
```

Reply:
Everything has been resolved now.
Thanks you